worksheet.rb

```diff
@@ -1,34 +1,144 @@
 ########################################################
 # Step 1: Establish the layers
 
-# In this section of the file, as a series of comments,
 # create a list of the layers you identify.
-# Which layers are nested in each other?
-# Which layers of data "have" within it a different layer?
-# Which layers are "next" to each other?
+#   Drivers, dates, fares, riders, and ratings
+#
+# The outermost layer is the entire table, the list of rides.
+#   Inside, 'next' to each other, will be the driver hashes. With driver and ride information.
+#   Inside the ride information will be another layer, with a list of rides.
+#   Inside the rides will be the ride details, with the rider, fare, date, and rating information running parallel.
 
 ########################################################
 # Step 2: Assign a data structure to each layer
-
-# Copy your list from above, and in this section
-# determine what data structure each layer should have
+#
+# The outermost layer is the entire table, the list of rides. Rides = {}
+#   Inside, 'next' to each other, will be the driver hashes. With driver and ride information. :DriverID, ride_info
+#     Inside the ride information will be another layer, with a list of rides. ride_info = [multiple rides]
+#       Inside the rides will be the ride details, with the rider, fare, date, and rating information running parallel. ride = {:RiderID, :$, :*}
 
 ########################################################
 # Step 3: Make the data structure!
 
-# Setup the entire data structure:
-# based off of the notes you have above, create the
-# and manually write in data presented in rides.csv
-# You should be copying and pasting the literal data
-# into this data structure, such as "DR0004"
-# and "3rd Feb 2016" and "RD0022"
+Rideshare = {
```

Comment:
Style: In the future constants (variables outside of methods that don't change) like this should be in `ALL_CAPS`.

```suggestion
RIDE_SHARE = {
```